internal/handlers/handlers.go

```diff
@@ -0,0 +1,30 @@
+package handlers
+
+import (
+	"context"
+	
+	"shortener/internal/storage"
+)
+
+type Storage interface {
+	Push(string, string) (string, error)
+	Get(string) (string, bool)
+	Users(string, string) ([]storage.Users, error)
```

Comment:
вообще в идеале бы я уехал на использование структуры в аргументе и/или возвращаемом значении. Тогда если нам надо будет расширить количество сохраняемых полей то достаточно будет добавить поле в структуру и не придется ходить по всем имплементациям и интерфейсам менять сигнатуры методов.
И пример к именованию 
```suggestion
	SaveUrl(model.URL) error
	GetURLByID(string) (model.URL, error)
	GetURLsByUserID(string, string) ([]model.URL, error)
```
пример для иллюстрации, понятно что он может не лечь на текущую архитектуру